KNOWN_ISSUES.md

```diff
@@ -0,0 +1,5 @@
+# Known Issues
+
+## Call Message API
+
+- On iOS, when sending a CallMessage with a payload that exceeds the maximum payload size, neither the `OutgoingCallMessage#sent` or `OutgoingCallMessage#failure` will fire.
```

Comment:
This will get fixed before GA and we have a ticket for it right? If so, no need to add into known issues.